db/fixtures/miq_product_features.yml

```diff
@@ -2966,6 +2966,11 @@
           :feature_type: admin
           :hidden: false
           :identifier: rbac_user_copy
+        - :name: Unlock
+          :description: Unlock a User
+          :feature_type: admin
+          :hidden: false
+          :identifier: rbac_user_unlock
```

Comment:
I would also remove this feature until we actually add it.  The way the code is written an unlock is not needed (because it will unlock in 2 minutes anyway)